pom.xml

```diff
@@ -55,7 +55,7 @@
         <reports.repository.url />
         <project.scm.id>github</project.scm.id>
         <!-- Must MANUALLY update this if the ddf/support project's version changes -->
-        <ddf.support.version>2.3.14</ddf.support.version>
+        <ddf.support.version>2.3.16</ddf.support.version>
```

Comment:
@paouelle, I see that this branch had been on version 2.3.14, not 2.3.15, of ddf-support. Are there any potential issues that might arise from 2.3.15 that need to be reflected here?

Reply:
If I look at what changes between 2.14 and 2.16, only the artifact enforcer was added. So you should be able to answer your own question 😄